2_globus.ipynb

```diff
@@ -20,17 +20,6 @@
     "* Explain how to find the Guest Collection"
```

Comment:
<p>should change the second paragraph to:</p><p> </p><blockquote>The method will return the manifest of the files that were copied and will also write a copy of the manifest to the local directory named &lt;dataset&gt;-manifest.json.</blockquote><p> </p><p>because we no longer return the URL</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>